toppra/solverwrapper/hot_qpoases_solverwrapper.py

```diff
@@ -268,12 +268,8 @@ def solve_stagewise_optim(self, i, H, g, x_min, x_max, x_next_min, x_next_max):
                 and np.all(0 >= self._l)
             ):
                 logger.fatal(
-                    "(0, 0) satisfies all constraints => error due to numerical errors.",
-                    self._A,
-                    self._lA,
-                    self._hA,
-                    self._l,
-                    self._h,
+                    f"(0, 0) satisfies all constraints => error due to numerical errors, \
+                        {self._A}, {self._lA}, {self._hA}, {self._l}, {self._h}"
```

Comment:
```suggestion
                    f"(0, 0) satisfies all constraints => error due to numerical errors, \
                        %s, %s, %s, %s, %s",  self._A, self._lA, self._hA, self._l, self._h

```

This would be slightly more efficient as it allows Python to do the string interpolation later.